src/renderers/shared/fiber/ReactFiberCompleteWork.js

```diff
@@ -161,6 +164,7 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
         // down its children. Instead, we'll get insertions from each child in
         // the portal directly.
       } else if (node.child !== null) {
+        node.child.return = node;
```

Comment:
How come we never bumped into this?

Reply:
It's always newly constructed in non-persistent mode.

Reply:
<img src="http://www.datagif.fr/reactionstory/images/wow.gif" width="50">

Reply:
At least, in sync mode it is?

Reply:
Even async mode resumes happen on a single tree. No "current" until after commit.